charts/capsule/Chart.yaml

```diff
@@ -21,7 +21,7 @@ sources:
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.0.19
+version: 0.0.20
```

Comment:
Let's avoid incrementing the Chart version since the proposed changes must be addressed when releasing a new Capsule release and would break up the Helm release during the clean-up.

Reply:
ok, reverted